Fixes building on sid.
(Or any setup where Ncurses is built using opaque types.)
This change will also be applied to the next upstream version.
--- a/src/printtext.cpp
+++ b/src/printtext.cpp
@@ -678,7 +678,9 @@
 static void
 do_indent(WINDOW *win, const int indent, int *insert_count)
 {
-	const attr_t attrs = win->_attrs;
+	attr_t attrs = 0;
+
+	(void) wattr_get(win, &attrs, NULL, NULL);
 
 	/* turn off all attributes during indentation */
 	(void) wattrset(win, A_NORMAL);
@@ -708,7 +710,7 @@
 static bool
 start_on_a_new_row(const ptrdiff_t sum, const WINDOW *win)
 {
-	return (sum < win->_maxx ? false : true);
+	return (sum < getmaxx(win) ? false : true);
 }
 
 /**
--- a/src/terminal.c
+++ b/src/terminal.c
@@ -69,8 +69,8 @@
 {
 	struct current_cursor_pos yx = { 0 };
 
-	yx.cury = win->_cury; /* row */
-	yx.curx = win->_curx; /* col */
+	yx.cury = getcury(win); /* row */
+	yx.curx = getcurx(win); /* col */
 
 	return yx;
 }
--- a/src/term-unix.h
+++ b/src/term-unix.h
@@ -35,7 +35,7 @@
 term_set_attr(WINDOW *win, attr_t at)
 {
 	if (win != NULL)
-		win->_attrs = at;
+		(void)wattrset(win, at);
 }
 
 #endif
